src/main/scala/org/allenai/plugins/DockerBuildPlugin.scala

```diff
@@ -129,6 +129,20 @@ object DockerBuildPlugin extends AutoPlugin {
     )
 
     ////////////////////////////////////////////////////////////////////////////////////////////////
+    // The following settings affect how the plugin behaves, but don't affect file or image output.
+    ////////////////////////////////////////////////////////////////////////////////////////////////
+
+    val verifyDockerfileIsStrict: SettingKey[Boolean] = Def.settingKey[Boolean](
+      "If true, the `verifyDockerfile` task will raise an error if the current Dockerfile does " +
+        "not match the generated one. Defaults to true."
+    )
+
+    val verifyDockerfileOnBuild: SettingKey[Boolean] = Def.settingKey[Boolean](
+      "If true, `dockerBuild` will depend on `verifyDockerfile`, and will not build if the " +
+        "Dockerfile is not up-to-date. Defaults to false."
```

Comment:
I think it's more correct to have this as a CI check, not a build-time check. I think it'll cause a lot of annoyances if it's happening on all builds.

Having a warning on all builds by default is good, but I was getting lazy and didn't want to add a third option . . .